src/types/TestImporter.ts

```diff
@@ -0,0 +1,22 @@
+import { MatchCategory } from './Match/MatchCategory';
+import { Match } from './Match/Match';
+
+//  Below is what my IDE auto generates for import
+// import { MatchCategory } from 'src/types/Match/MatchCategory'
+// import { Match } from 'src/types/Match/Match'
+
```

Comment:
When using IDE completion, the above is what it automatically generates in IntelliJ. Currently I have to manually type it (as shown in lines 1/2) - which through investigation is the root cause of this issue. Any ideas?

Reply:
https://stackoverflow.com/questions/55198502/using-eslint-with-typescript-unable-to-resolve-path-to-module Is what is meant to fix it, but it doesn't appear to be working for me

Reply:
Errors are as below:

Typescript compile error: 'Cannot find module for 'src/types/Match....'
ESLint: 'Unable to resolve path to module 'src/types/Match....'

Reply:
Related CRA issues:
facebook/create-react-app#5692
facebook/create-react-app#5645
facebook/create-react-app#5118